sql/moz-fx-data-marketing-prod/acoustic/contact_v1/query.sql

```diff
@@ -0,0 +1,155 @@
+      ) AS sub_mozilla_technology,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_mozillians_nda AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_mozillians_nda,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_firefox_accounts_journey AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_firefox_accounts_journey,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_knowledge_is_power AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_knowledge_is_power,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_take_action_for_the_internet AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_take_action_for_the_internet,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_test_pilot AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_test_pilot,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_firefox_news AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_firefox_news,  -- 2 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_about_mozilla AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_about_mozilla,  -- 13436 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_apps_and_hacks AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_apps_and_hacks,  -- 12988 nulls  // boolean
+      CAST(
+        COALESCE(CAST(sub_rally AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_rally,  -- 13641 nulls  // boolean
+      CAST(
+        COALESCE(CAST(NULLIF(sub_firefox_sweepstakes, '') AS INTEGER), 0) AS BOOLEAN
+      ) AS sub_firefox_sweepstakes,  -- 14226 nulls  // boolean
+      LOWER(
+        NULLIF(vpn_waitlist_geo, '')
+      ) AS vpn_waitlist_geo,  -- 1221463 nulls, contains case diffences  // no_value is a bad choice of default, should this just be null?
+      SPLIT(
+        NULLIF(vpn_waitlist_platform, ''),
+        ","
+      ) AS vpn_waitlist_platform,  -- 1230937 nulls, single field contains multiple values comma separated, should this be a repeated field?  // repeated string field
+      relay_waitlist_geo,  -- # missing field, need to verify, maybe it's related to reusing old connector
+      RECIPIENT_ID AS recipient_id,  -- Acoustic system field
+      FORMAT_FXA_DATE(
+        Last_Modified_Date
+      ) AS acoustic_last_modifled_date,  -- Acoustic system field
+      @submission_date AS job_date,
+      CURRENT_DATETIME() AS processed_at,
+    FROM
+      `moz-fx-data-bq-fivetran.acoustic_sftp.contact_export`
+  )
+
+  , YESTERDAY_LOADED_CONTACTS AS (
+      SELECT * EXCEPT(job_date, processed_at)
+      FROM `moz-fx-data-marketing-prod.acoustic.contact_v1`  -- possible that this may not exist on first run
+      WHERE job_date = DATE_SUB(@submission_date, INTERVAL 1 DAY)
+  )
+
+
+  SELECT * FROM JUST_LOADED_CONTACTS
+  UNION ALL
+  SELECT
+    *,
+    @submission_date AS job_date,
+    CURRENT_DATETIME() AS processed_at,
+  FROM YESTERDAY_LOADED_CONTACTS
+  WHERE email_id NOT IN (SELECT DISTINCT email_id FROM JUST_LOADED_CONTACTS);
+
+  TRUNCATE TABLE `moz-fx-data-bq-fivetran.acoustic_sftp.contact_export`;
```

Comment:
Maybe add a comment here saying why it's necessary to truncate the source table after running the ETL? A link is probably fine

Reply:
Added a comment to provide an explanation. Please let me know if it provides enough context.